src/Tickets/Seating/app/blockEditor/store/index.js

```diff
@@ -104,7 +111,26 @@ const store = createReduxStore(storeName, {
 					},
 					seatTypesByPostId: {
 						...state.seatTypesByPostId,
-						[ticketPostId]: action.seatTypeId,
+						[ticketPostId || action.clientId]: action.seatTypeId,
+					},
+				};
+			case 'SET_TICKET_SEAT_TYPE_BY_POST_ID':
+				const ticketId = getTicketIdFromCommonStore(action.clientId);
+
+				const { seatTypesByPostId, seatTypesByClientId } = state;
+
+				const seatTypeId =
+					seatTypesByClientId[action.clientId] ||
+					seatTypesByPostId[action.clientId] ||
+					seatTypesByPostId[ticketId];
+
+				delete seatTypesByPostId[action.clientId];
```

Comment:
Why remove this and not **update** it?

Reply:
We could also skip setting a value inside `seatTypesByPostId` until we have a ticket id.